src/katgpucbf/xbgpu/engine.py

```diff
@@ -969,14 +965,14 @@ class XBEngine(DeviceServer):
         Hostname or IP on which to listen for KATCP C&M connections.
     katcp_port
         Network port on which to listen for KATCP C&M connections.
-    adc_sample_rate_hz
+    adc_sample_rate
         Sample rate of the digitisers in the current array. This value is
         required to calculate the packet spacing of the output heaps. If it is
         set incorrectly, the packet spacing could be too large causing the
         pipeline to stall as heaps queue at the sender faster than they are
         sent.
     bandwidth_hz
```

Comment:
Let's take out the `_hz` here too (and anywhere else it turns up in variable names).